spirv_glsl.cpp

```diff
@@ -10265,7 +10292,28 @@ void CompilerGLSL::emit_instruction(const Instruction &instruction)
 		break;
 
 	case OpTranspose:
-		GLSL_UFOP(transpose);
+		if (options.version < 120) // Matches GLSL 1.10 / ESSL 1.00
```

Comment:
Isn't this the same as `is_legacy()`?

Reply:
Nope, because `is_legacy()` includes GLSL 1.20, but that does have `transpose()`